pytransform3d/batch_rotations.py

```diff
@@ -38,6 +38,52 @@ def norm_vectors(V, out=None):
     return out
 
 
+def norm_axis_angles(a):
+    """Normalize axis-angle representation.
+
+    Parameters
+    ----------
+    a : array-like, shape (..., 4)
+        Axis of rotation and rotation angle: (x, y, z, angle)
+
+    Returns
+    -------
+    a : array, shape (..., 4)
+        Axis of rotation and rotation angle: (x, y, z, angle). The length
+        of the axis vector is 1 and the angle is in [0, pi). No rotation
+        is represented by [1, 0, 0, 0].
+    """
+    a = np.asarray(a)
+
+    # Handle the case of only one axis-angle instance
+    only_one = a.ndim == 1
+    a = np.atleast_2d(a)
+
+    angles = a[..., 3]
+    norm = np.linalg.norm(a[..., :3], axis=-1)
```

Comment:
You probably wanted to use the variable:

```suggestion
    norm = np.linalg.norm(angles, axis=-1)
```

Reply:
No, we want to use the axes, not the angles, but I had to try it to see the mistake, too. :D